CMakeLists.txt

```diff
@@ -77,12 +79,15 @@ if(NOT EXISTS "${GLOW_THIRDPARTY_DIR}/onnx")
   message(FATAL_ERROR "No onnx git submodule. Run: git submodule update --init --recursive")
 endif()
 
-add_subdirectory(tests/googletest/)
-
 add_subdirectory(lib)
-add_subdirectory(examples)
 add_subdirectory(tools)
-add_subdirectory(tests)
+if (GLOW_BUILD_EXAMPLES)
+  add_subdirectory(examples)
+endif()
+if (GLOW_BUILD_TESTS)
+  add_subdirectory(tests/googletest)
```

Comment:
there are couple of things at the very end:
```
 # Fetch the dependencies for all the tests.
 get_property(GLOW_TEST_DEPENDS GLOBAL PROPERTY GLOW_TEST_DEPENDS)
 
 add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND}
                     DEPENDS ${GLOW_TEST_DEPENDS} USES_TERMINAL)
```
Just move it here behind the if (GLOW_BUILD_TESTS).

Reply:
Done